### PR TITLE
Added the tzlocal python module to the install script

### PR DESCRIPTION
**Pull request by pmeenan**

### CODE REVIEW

**Review by Federico-Mulas — CHANGES_REQUESTED** [review]
Sorry I had this change locally but didn't post the PR for it. My bad.

Should the macos.sh script have the same change too?

File: debian.sh [hunk 1]
```diff
@@ -302,7 +302,7 @@ fi
 #**************************************************************************************************
 # Python Modules
 #**************************************************************************************************
-until sudo pip3 install dnspython monotonic pillow psutil requests tornado wsaccel brotli fonttools selenium future usbmuxwrapper
+until sudo pip3 install dnspython monotonic pillow psutil requests tornado wsaccel brotli fonttools selenium future usbmuxwrapper tzlocal
```
Comment (FedericoMulas) [inline debian.sh:305]: https://github.com/catchpoint/WebPageTest.agent/pull/638/files#diff-292d079fe725eed06ead69e3675d638d8bda0c4b65aae36ebde665c7ed52a2beR20

The new logging has also pytz as dependency
```suggestion
until sudo pip3 install dnspython monotonic pillow psutil requests tornado wsaccel brotli fonttools selenium future usbmuxwrapper pytz tzlocal
```

**Review by FedericoMulas — CHANGES_REQUESTED** [review]
> Sorry I had this change locally but didn't post the PR for it. My bad.
> 
> Should the macos.sh script have the same change too?

And sorry I also messed up commenting with my non catchpoint account...

**Comment by pmeenan** [comment]
Thanks. Added `pytz` and also updated the macos script

**Review by FedericoMulas — APPROVED** [review]
Looks good to me!

**Pull request merged**